_kbarticles/2022-05-05-creating-postgresql-databases-from-shared-rds.md

```diff
@@ -0,0 +1,88 @@
+---
+layout: post
+title: "Creating PostgreSQL databases from a shared RDS instance using postgres-tinsmith"
+date: May 5, 2022
+excerpt: How to use postgres-tinsmith service broker to quickly create PostgreSQL databases from a shared RDS instance
+---
+
+The cloud.gov team [recently deprecated the shared database instance plans]({{ site.baseurl }}{% link _posts/2022-04-25-deprecation-notice.md %}) of our platform.
+
+One of the major benefits of the shared database instance plans was the ability to very quickly create new
+databases, facilitating rapid application development. Unlike dedicated instance plans, which each provision a
+separate RDS database instance and take a while to create, shared instance plans re-used the same RDS database instance and created new databases within the same instance, which completed almost instantly.
+
+This article will cover how to replicate the behavior of the deprecated shared instance plans using an 
+open-source service broker known as [`postgres-tinsmith`](https://github.com/markdboyd/cf-postgres-tinsmith).
+
+Please note that while this approach has been tested manually, the `postgres-tinsmith` repo is not
+actively maintained by the cloud.gov team, so this documentation is provided as a **proof-of-concept only and does not offer any guarantees of production readiness**.
+
+## 1. Create a dedicated RDS service
+
+View our available dedicated RDS instance plans for PostgreSQL:
+
+```shell
+cf marketplace -e aws-rds
+```
+
+Create a PostgreSQL service using one of our dedicated instance plans. For example:
+
+```shell
+cf create-service aws-rds micro-psql <dedicated-db-name>
+```
+
+## 2. Deploy and configure `postgres-tinsmith`
+
+Follow [the instructions in the repo README](https://github.com/markdboyd/cf-postgres-tinsmith) to create and configure the `postgres-tinsmith` service broker.
+
+Make sure to use the `<dedicated-db-name>` created in step 1 as the database service name to bind to
+`postgres-tinsmith`.
+
+## 3. Create a database
+
+Once `postgres-tinsmith` is deployed and configured, create a database using the new `shared` plan of the
+`postgres` service:
+
+```shell
+cf create-service postgres shared <db-name>
+```
+
+## 4. Bind database to your application
+
+### Binding using `manifest.yml`
```

Comment:
I'd skip going into the manifest method here and just refer to [the upstream documentation on how to bind a service in a manifest](https://docs.cloudfoundry.org/devguide/deploy-apps/manifest-attributes.html#services-block) after you explain how to do it imperatively with the CLI commands.